app/metrics.yaml

```diff
@@ -137,6 +137,7 @@ events:
           show_search_shortcuts, open_links_in_a_private_tab (bug in
           implementation https://github.com/mozilla-mobile/fenix/issues/7384),
           pref_key_sync_logins, pref_key_sync_bookmarks, pref_key_sync_history,
+          pref_key_show_voice_search,
```

Comment:
Hey @sblatz, do you think these pref states should be moved to the metrics ping? These are different, someone actively toggling a pref vs what the value of the pref is - do we care how often a pref is toggled, versus how many people have it enabled/disabled?

Reply:
Unclear. My understanding was that @vesta0 wanted to know when users went in to settings to disable it. 

@vesta0 would you prefer this be sent in the baseline ping? e.g. "user has voice search disabled, has open links in private tab enabled, etc.)? 

I agree this seems more useful but I think we also wanted to know if people were aware they could even *change* these settings.

Reply:
As we discussed, let's merge this as is for now. Sounds like there may be some more clean up data science and I need to do on these to get better data from them, but that's a larger project!